app/javascript/src/components/form/Derivatives.svelte

```diff
@@ -0,0 +1,84 @@
+<script>
+  import Tags from './Tags.svelte';
+  import { onMount } from 'svelte';
+
+  let showDerivative = false;
+  let maxCodes = 5;
+  export let currSources = [];
```

Comment:
Small thing, but I try to stay away from abbreviations, especially for props. It's easier to remember a word than it is an abbreviation, in my opinion.